include/swift/IDE/CodeCompletionResult.h

```diff
@@ -334,6 +367,8 @@ class ContextFreeCodeCompletionResult {
   CodeCompletionOperatorKind KnownOperatorKind : 6;
   static_assert(int(CodeCompletionOperatorKind::MAX_VALUE) < 1 << 6, "");
 
+  CodeCompletionMacroRoles DeclMacroRoles;
```

Comment:
Shouldn’t this just be `MacroRoles`?

Reply:
Yeah, given that they now include freestanding :(. Originally this was just attached.